### PR TITLE
edit

### PR DESCRIPTION
**Pull request by sffrench**
Just some minor fixes for you Jaime. I hope this workflow is ok.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -8,6 +8,7 @@ This demo uses Red{nbsp}Hat OpenShift Data Science (RHODS) to train and deploy a
 Sounds are classified into categories such as `dog`.
 
 You will deploy the model in two ways:
+// TechEditor:[style] "will" as a future tense is generally avoided. Are te two ways two different options? if yes then, "you can" could be used here.
```
Comment (jramcast) [inline README.adoc]: In the context of this demo, they are not options (one or the other), but two different steps, performed one after the other.

In general, a user can deploy a model in many ways.
This exercise covers two of these ways.

What about this?

```suggestion
The demo instructions guide you to deploy the model in two ways:
```

File: README.adoc [hunk 2]
```diff
@@ -414,6 +415,7 @@ This container includes the model, so you can use it to perform inference at the
 11. *Deploy a web application that uses the deployed model via the Model Service REST API*.
 
 a. From the developer perspective of the OpenShift web console, and click btn:[+Add], then click btn:[Import YAML].
+// TechEditor:[style] This sentence does not make sense. I think the "and" is misplaced. If you remove the "and" then the sentence makes sense. I wasnt sure if you had more intention to introduce something here so I have lf it up to you to take a look at.
```
Comment (jramcast) [inline README.adoc]: Yes, I think I rewrote the sentence at some point and forgot the remove the "and".

```suggestion
a. From the developer perspective of the OpenShift web console, click btn:[+Add], then click btn:[Import YAML].
// TechEditor:[style] This sentence does not make sense. I think the "and" is misplaced. If you remove the "and" then the sentence makes sense. I wasnt sure if you had more intention to introduce something here so I have lf it up to you to take a look at. 
```

File: README.adoc [hunk 3]
```diff
@@ -438,8 +440,8 @@ e. Click btn:[+Add], then click btn:[Import YAML].
 f. In the form, paste the contents of the https://github.com/jramcast/rhods-sound-classification/blob/main/apps/web/front/app.yaml file.
 This manifest deploys a container image that corresponds to the application at `apps/web/front`.
 
-g. Replace the value of the `BACKEND` environment variable with the route URL of the backend.
-The front end requires this value to connect to the back end API.
+g. Replace the value of the `BACKEND` environment variable with the route URL of the back-end.
```
Comment (jramcast) [inline README.adoc]: Should it be "back end" when used as a noun?
```suggestion
g. Replace the value of the `BACKEND` environment variable with the route URL of the back end.
```

**Review by jramcast — APPROVED** [review]
Thank you for the fixes @sffrench ! I left a few suggestions in response to your comments. 
If you are ok with those, feel free to go ahead an merge.

File: README.adoc (hunk 1 above)
Comment (sffrench) [reply README.adoc]: perfect.

File: README.adoc (hunk 3 above)
Comment (sffrench) [reply README.adoc]: @jramcast 
n. Two words. Refers to software that performs the final stages of a process, or to tasks that are not visible to the user. For example, "each back end provides a set of calls".

Use the one-word form only if it is part of the established product terminology, for example "Mobile Backend-as-a-Service", and when it is not being used to describe a generic process.

File: README.adoc (hunk 1 above)
Comment (sffrench) [inline README.adoc]:
```suggestion
```

File: README.adoc (hunk 2 above)
Comment (sffrench) [inline README.adoc]:
```suggestion
```

**Review by sffrench — COMMENTED** [review]
Remove tech edit comments.

**Pull request merged**